client/js/recurring_donations/edit/index.es6

```diff
@@ -363,7 +361,10 @@ const paydateForm = state =>
       }
     })
   , h('br')
-  , button(R.pick(['loading$', 'error$'], state))
+  , button({
+    ...(state.loading$ && { loading$: state.loading$ }),
+    ...(state.error$ && { error$: state.error$ }),
```

Comment:
This is another place that I'm unsure on whether this is correct. When a boolean is false, what happens?

Reply:
it should work, commented on the other one, but as with there, I'll just use `pick` from lodash, it's more readable.